sw/host/cryptotest/testvectors/parsers/wycheproof_ecdh_parser.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import json
+import jsonschema
+import logging
+import sys
+
+from Crypto.Util.asn1 import DerSequence, DerObjectId, DerBitString
+
+# Mapping from the curve names used by Wycheproof to those used by cryptotest
+EC_NAME_MAPPING = {
+    "secp256r1": "p256",
+    "secp384r1": "p384",
+}
+
+# Object ID for algorithm `eccPublicKey` (https://oidref.com/1.2.840.10045.2.1)
+EccPublicKeyOid = "1.2.840.10045.2.1"
+
+EC_OID_MAPPING = {
+    # https://oidref.com/1.2.840.10045.3.1.7
+    "secp256r1": "1.2.840.10045.3.1.7",
+    # https://oidref.com/1.3.132.0.34
+    "secp384r1": "1.3.132.0.34",
+}
+
+EC_COORDINATE_SIZE_MAPPING = {
+    "secp256r1": 32,
+    "secp384r1": 48,
+}
+
+# A small number of tests include invalid OIDs in the DER that
+# Crypto.Util.asn1 fails to reject, so we manually exclude those test
+# cases. The test case IDs match what is in
+# sw/vendor/wycheproof/testvectors/ecdh_secp256r1_test.json at the
+# time of this writing. The IDs differ in the most up-to-date
+# wycheproof master branch.
+PUBLIC_KEY_BLACKLIST = [
```

Comment:
The name of this list could be a bit more descriptive. Maybe something like `INVALID_OID_PUBKEYS`. And also [prefer](https://developers.google.com/style/word-list#blacklist) "denylist" or "blocklist" to "blacklist".